cli/src/generate/templates/build.rs

```diff
@@ -8,6 +8,9 @@ fn main() {
     c_config.file(&parser_path);
     println!("cargo:rerun-if-changed={}", parser_path.to_str().unwrap());
 
+    #[cfg(target_os = "windows")]
+    c_config.flag("/utf-8");
```

Comment:
```suggestion
    #[cfg(target_env = "msvc")]
    c_config.flag("-utf-8");
```

The flag should not be added in mingw and cc-rs uses dashes.

Reply:
`msvc` not `windows`

Reply:
oops, sorry